bullet_train/bullet_train.py

```diff
@@ -98,6 +103,58 @@ def get_value(self, feature_name, identity=None):
         else:
             return None
 
+    def get_trait(self, trait_key, identity):
+        """
+        Get value of given trait for the identity of an environment.
+
+        :param trait_key: key of trait to determine value of (must match 'ID' on bullet-train.io)
+        :param identity: application's unique identifier for the user to check feature state
+        :return: Trait value. None otherwise.
+        """
+        if not trait_key:
+            return None
+
+        if not identity:
+            return None
+
+        data = self._get_flags_response(None, identity)
+
+        traits = data['traits']
+        for trait in traits:
+            if trait.get('trait_key') == trait_key:
+                return trait.get('trait_value')
+
+    def set_trait(self, trait_key, trait_value, identity):
+        """
+        Set value of given trait for the identity of an environment. Note that this will lazily create
+        a new trait if the trait_key has not been seen before for this identity
+
+        :param trait_key: key of trait
+        :param trait_value: value of trait
+        :param identity: application's unique identifier for the user to check feature state
+        """
+        if not trait_key:
+            return None
+
+        if not trait_value:
+            return None
+
+        if not identity:
+            return None
+
+        identifier = {}
+        identifier['identifier'] = identity
+        payload = {}
+        payload['identity'] = identifier
+        payload['trait_key'] = trait_key
+        payload['trait_value'] = trait_value
+        
+        r = requests.post(self.traits_endpoint, 
+                      json=payload, 
+                      headers=self._generate_header_content())
+        logging.warning(r)
```

Comment:
Use `logger` instead of `logging`. 

Although I'm not sure why this is even here? Left over debugging? If you want to show the response, use `logger.info` not warning.